pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors"}]
```

Comment:
I can add a link here to https://github.com/dask/dask-image/graphs/contributors
What do we think?